docs/index.html

```diff
@@ -3128,6 +3130,25 @@ <h3 id="viewing-console-logs">Viewing Console Logs</h3>
 </code></pre>
     </div>
   </div>
+</div><div class="row">
+  <div class="row-height">
+    <div class="col-md-5 col-sm-12 col-height  docs-primary">
+      <h3 id="viewing-bundle-logs">Viewing Bundle Logs</h3>
+    </div>
+    <div class="col-md-7 col-sm-12 col-height is-empty docs-code">
+      
```

Comment:
Nit: `git check` doesn't like the trailing whitespace here:

```console
$ git show --oneline --check origin/pr/291
5fac9b5 (origin/pr/291, origin/feature-bundle-logs) Updating docs.
docs/index.html:3139: trailing whitespace.
+      
```

Reply:
Thanks for the tip @wking ! That's part of https://github.com/zapier/litdoc so maybe worth adding something there?